src/main/java/io/vlingo/symbio/store/object/jdbc/JDBCObjectStoreEntryReaderActor.java

```diff
@@ -77,16 +90,45 @@ public void close() {
     return completes().with(name);
   }
 
+  private List<Entry<String>> readIds(List<Long> ids) {
```

Comment:
@pflueras Can we move private methods to the bottom of the class? I find it helpful to have the public interface at the top, followed by protected, and then finally private. I also usually put each second in alphabetical order. It helps to browse through the code.

Reply:
Totally agree 👍. Implemented.